pwa/src/components/deleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import "./deletemodal.css";
+import { Modal } from "@conductionnl/nl-design-system";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+interface DeleteModalProps {
+  handleDelete: any;
+  handleId: any;
+  optionalLength?: any;
+}
+
+const DeleteModal: React.FC<DeleteModalProps> = ({ handleDelete, handleId, optionalLength }) => {
+  return (
+    <div className="DeleteModal">
+      <Modal
+        title="Warning!"
+        id={`deleteModal${handleId}`}
+        body={function () {
+          return (
+            <>
+              {optionalLength === 1 ? (
+                <p>
+                  Are you sure you want to delete {handleId}? <br></br>
+                  <b>If you delete this translation you wil also delete the Translation Table!</b>
+                </p>
+              ) : (
+                <p>Are you sure you want to delete {handleId}?</p>
+              )}
```

Comment:
We're now duplicating the `<p>Are you sure you want to delete {handleId}?</p>` part. An easier (and better readable) way of doing this would be:

```
<p>
  Are you sure you want to delete {handleId}? <br></br>
  { optionalLength === 1 && <b>If you delete this translation you wil also delete the Translation Table!</b> }
</p>
```

This halves the amount of code we have to write, and removes a hard to read (and hard to maintain) ternary (`? :`) statement.